Src/FluentAssertions/Common/TypeExtensions.cs

```diff
@@ -221,143 +218,23 @@ public static FieldInfo FindField(this Type type, string fieldName)
 
     public static IEnumerable<MemberInfo> GetNonPrivateMembers(this Type typeToReflect, MemberVisibility visibility)
```

Comment:
FYI, If the return type is changed to `MemberInfo[]` then `DefaultValueFormatter.GetMembers` can have it's `ToArray()` removed.

Reply:
Nice catch